src/lib/lodashFp.ts

```diff
@@ -0,0 +1,28 @@
+import { Extendable } from '../../types/utils'
+
+const _: Extendable<{}> = {
+  capitalize: require('lodash/fp/capitalize'),
+  compact: require('lodash/fp/compact'),
+  difference: require('lodash/fp/difference'),
+  each: require('lodash/fp/each'),
+  escapeRegExp: require('lodash/fp/escapeRegExp'),
+  filter: require('lodash/fp/filter'),
+  flow: require('lodash/fp/flow'),
+  isFunction: require('lodash/fp/isFunction'),
+  isNil: require('lodash/fp/isNil'),
+  isObject: require('lodash/fp/isObject'),
+  isPlainObject: require('lodash/fp/isPlainObject'),
+  keys: require('lodash/fp/keys'),
+  map: require('lodash/fp/map'),
+  max: require('lodash/fp/max'),
+  memoize: require('lodash/fp/memoize'),
+  min: require('lodash/fp/min'),
+  pick: require('lodash/fp/pick'),
+  sortBy: require('lodash/fp/sortBy'),
+  startCase: require('lodash/fp/startCase'),
+  sum: require('lodash/fp/sum'),
+  take: require('lodash/fp/take'),
+  trim: require('lodash/fp/trim'),
+}
```

Comment:
We should use only ES imports as we have ES modules build

Reply:
removed